App/index.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-cycle */
+export { DispatchAction } from './store/reducer'
+
+export { InfoTextBox, CheckBoxRow } from './components'
+
+export { default as Button, ButtonType } from './components/buttons/Button'
+
+export { testIdWithKey } from './utils/testable'
+
+export { default as ConnectionModal } from './components/modals/ConnectionModal'
```

Comment:
In line with previous comments we can simply import default modules using:
`import <module> from <path>`